client/src/store/new-request-module.ts

```diff
@@ -245,6 +245,13 @@ export class NewRequestModule extends VuexModule {
         `Reported separately as Corporate tax`,
         `Has name protection in BC`
       ],
+      mveBlurbs: [
+        `A company that may have one or more people who own shares with some personal responsibility for
+        debt and liabilities.`,
```

Comment:
I think "debt" should be plural (as on line 281).

Reply:
for sure!